java/src/jmri/jmrit/roster/UpdateDecoderDefinitionAction.java

```diff
@@ -51,16 +51,21 @@ public void actionPerformed(ActionEvent e) {
                 if (decoder.getReplacementFamily() != null || decoder.getReplacementModel() != null) {
                     log.info("   Recommended replacement is family \"" + decoder.getReplacementFamily() + "\" model \"" + decoder.getReplacementModel() + "\"");
                 }
-                replacementFamily = (decoder.getReplacementFamily() != null) ? decoder.getReplacementFamily() : replacementFamily;
-                replacementModel = (decoder.getReplacementModel() != null) ? decoder.getReplacementModel() : replacementModel;
+                replacementFamily = (decoder.getReplacementFamily() != null) ? decoder.getReplacementFamily() : null;
```

Comment:
(This should be a comment on line 55, but the same applies for line 54 with Family instead of Model)